client/outfits.lua

```diff
@@ -50,28 +50,36 @@ function LoadJobOutfit(oData)
         SetPedComponentVariation(ped, 6, data["shoes"].item, data["shoes"].texture, 0)
     end
 
-    -- Bag
```

Comment:
Doesn't this cause the outfits to not have a bag anymore?

Reply:
I didn't test it because my script takes control of the backpacks on the server. However, I will test it without it to see what happens. But, I believe that shouldn't remove bags.

Reply:
Yeah but what I mean is that this will not work anymore with the change above: https://github.com/iLLeniumStudios/illenium-appearance/blob/main/shared/config.lua#L610

As the table for the outfits goes through this event, it would not set the bag for that outfit and will keep whatever the user is wearing already

Reply:
I tested it on qbcore load/logout and restarting the script works fine.

Reply:
@swkeep can you try setting a different bag in the outfits and then see if putting on that outfit applies that bag? I think that is the case that I'm trying to make sure, is working

Reply:
@swkeep Nevermind. It should work, I didn't check the full file and it was just a duplicate, my bad